Script/char.dat

```diff
@@ -5332,6 +5403,7 @@ necromancer
   CanRead = true;
   CanBeGenerated = true;
   ClassStates = INFRA_VISION;
+  ConsumeFlags = CT_MEAT|CT_LIQUID|CT_PROCESSED|CT_BONE; /* Necromancers eat vegetarians, not vegetables! */
```

Comment:
TODO: Add Exalted Necromancer (otherwise normal necromancer who just ate a bunch of ommel bone)